components/installer/README.md

```diff
@@ -111,7 +111,19 @@ Example:
 ../../installation/scripts/create-cr.sh --output installer-cr.yaml --url {URL TO KYMA TAR GZ} --version 0.0.1
 ```
 
-To run the installer with the Azure Broker enabled, specify the Azure credentials in the Custom Resource file. Edit the file by providing the following properties along with their values in the **spec** definition: `az_subscription_id`, `az_tenant_id`, `az_client_id` and `az_client_secret`.
+### Enabling Azure Broker
+
+To run the Kyma with the Azure Broker enabled, mark `azure-broker` subcomponent as enabled using `manage-component.sh` script or manually. Specify the Azure credentials as the environment variables providing the following variables along with their values encoded as base64 strings: `AZURE_BROKER_SUBSCRIPTION_ID`, `AZURE_BROKER_TENANT_ID`, `AZURE_BROKER_CLIENT_ID` and `AZURE_BROKER_CLIENT_SECRET`.
```

Comment:
1. mark `azure-broker` subcomponent as enabled using `manage-component.sh` script or manually. -> ``mark the `azure-broker` subcomponent as enabled either using the `manage-component.sh` script or manually.``
2. `` `AZURE_BROKER_CLIENT_ID`, and `AZURE_BROKER_CLIENT_SECRET`.`` (serial comma was missing, sorry)

Reply:
Done.